docs/examples/custom_naming/sql_ci_no_collision.py

```diff
@@ -0,0 +1,34 @@
+from typing import ClassVar
```

Comment:
these extra files will not show up in the docs, maybe this is ok for now, but we might need to adapt this somehow?

Reply:
heh true but people may click in the link and they will go to github. LGTM, for now